source/lmp/fix_dplr.h

```diff
@@ -71,6 +71,12 @@ class FixDPLR : public Fix {
   std::vector<double> efield_fsum, efield_fsum_all;
   int efield_force_flag;
   void get_valid_pairs(std::vector<std::pair<int, int> > &pairs);
+  int varflag;
+  char *xstr, *ystr, *zstr, *estr, *pstr;
```

Comment:
it seems that estr and pstr are not used.

Reply:
Yes. I deleted them.